prow/plugins.yaml

```diff
@@ -433,6 +434,20 @@ plugins:
 
   GoogleCloudPlatform/k8s-multicluster-ingress:
   - trigger
+  
+  client-go/unofficial-docs:
```

Comment:
Consider enabling these everywhere in the client-go org? (just change to `client-go:`)